specification/vmware/resource-manager/readme.md

```diff
@@ -239,10 +254,43 @@ suppressions:
     # https://azure.github.io/autorest/extensions/#x-ms-long-running-operation-options
     from: vmware.json
 
-  # just warnings
-  # - code: IgnoredPropertyNextToRef
-  # - code: LroLocationHeader
-  #   from: vmware.json
+  - code: XmsClientName
+    reason: false positives
+    from: vmware.json
+
+  - code: XmsClientNameParameter
+    reason: this warning is false postive
+    from: vmware.json
+
+  - code: docLinkLocale
+    reason: false positives
+    from: vmware.json
+
+  - code: PropertiesTypeObjectNoDefinition
+    reason: false positives
+    from: vmware.json
+
+  - code: LatestVersionOfCommonTypesMustBeUsed
+    reason: this warning is TODO
+    from: vmware.json
+
+  - code: AvoidAdditionalProperties
+    reasons: this is how TypeSpec models records
```

Comment:
>records [](http://example.com/codeflow?start=41&length=7)

what records are we talking about here?

Reply:
Probably related to https://github.com/Azure/azure-sdk-tools/issues/7803 .

Reply:
This suppression has been removed.